frontend/packages/kubevirt-plugin/integration-tests-cypress/support/index.ts

```diff
@@ -0,0 +1,75 @@
+import { ProvisionSource } from '../../integration-tests/tests/utils/constants/enums/provisionSource';
+
+export * from '../../../integration-tests-cypress/support';
+
+export const KUBEVIRT_STORAGE_CLASS_DEFAULTS = 'kubevirt-storage-class-defaults';
+
+declare global {
+  namespace Cypress {
+    interface Chainable<Subject> {
+      deleteResource(resource, ignoreNotFound?: boolean): void;
+      createResource(resource): void;
+      createDataVolume(name: string, namespace: string): void;
+      dropFile(fileName: string, inputSelector: string): void;
+    }
+  }
+}
+
+Cypress.Commands.add('deleteResource', (resource, ignoreNotFound = true) => {
+  const kind = resource.kind === 'NetworkAttachmentDefinition' ? 'net-attach-def' : resource.kind;
+  cy.exec(
+    `kubectl delete --ignore-not-found=${ignoreNotFound} -n ${resource.metadata.namespace} --cascade ${kind} ${resource.metadata.name}`,
+  );
+});
+
+Cypress.Commands.add('createResource', (resource) => {
+  cy.exec(`echo '${JSON.stringify(resource)}' | kubectl create -f -`);
+});
+
+Cypress.Commands.add('createDataVolume', (name: string, namespace: string) => {
```

Comment:
Can we put the dv json in fixture and just call `createResource` to do it?

Reply:
moved data volume to fixture. But I still need this command because I need to change the default name/namespace/access and volume modes. The command however uses `createResource`